src/components/DateRangePicker.jsx

```diff
@@ -391,7 +391,7 @@ class DateRangePicker extends React.PureComponent {
 
     if (withPortal || withFullScreenPortal || appendToBody) {
       return (
-        <Portal>
+        <Portal key="day-picker-portal">
```

Comment:
this isn't rendered in an array, so the key has no effect. why is this here?

Reply:
See below